make-release.sh

```diff
@@ -67,6 +66,12 @@ if [[ "${BASEBRANCH}" != "${BRANCH}" ]]; then
 fi
 
 apply_files_edits () {
+  read THEIA_VERSION < THEIA_VERSION
+  if [[ ! ${THEIA_VERSION} ]]; then
+    echo "THEIA_VERSION file is not found"; echo
```

Comment:
I wonder if, to Florent's point, there's value in being able to override the value in the file with `--theia-version` flag?

Reply:
it can be updated later if we'll need it